lib/fizzy/execute.cpp

```diff
@@ -1429,12 +1429,13 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
         }
         case Instr::i64_extend_i32_s:
         {
+            // FIXME: Add more tests and make sure this is correct.
```

Comment:
Do you want to add tests first before this PR?

Reply:
Done in #680 ?

Reply:
Yes, although this PR will extend the tests anyway by polluting the Value with some garbage (not possible before).